### DISCUSSION TITLE
Welcome to statements-on-tools Discussions!

### DISCUSSION THREAD

**Discussion by Lextuga007** [nhs-r-community/statements-on-tools#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wbryant** [comment]
Thanks this seems like a good place - I usually think most of my ideas are too poorly formed to be issues when I first write them down!
